core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -361,7 +361,7 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
   def adminRegisterBillingProject(xfer: RawlsBillingProjectTransfer): Future[PerRequestMessage] = {
     asFCAdmin {
       val billingProjectName = RawlsBillingProjectName(xfer.project)
-      val project = RawlsBillingProject(billingProjectName, CreationStatuses.Ready, Option(RawlsBillingAccountName("billingAccounts/00293C-5DEA2D-6887E7")), None)
+      val project = RawlsBillingProject(billingProjectName, CreationStatuses.Ready, Option(adminRegisterBillingAccountId), None)
```

Comment:
I never noticed that this endpoint existed, so in addition to reviewing your changes, I'm trying to wrap my head around what this api is doing.  

It is for transferring ownership of a Billing Project to a new user, right?  And for some reason, we expect to be able to ask the new owner for their email address and a bearer token, but we don't ask them for the Billing Account they want set?  Instead, we just default it to some Billing Account that we own and pay for and then expect the user to go in and update it?  

Wouldn't it be better if we just made the Billing Account part of `RawlsBillingProjectTransfer` and add that to the request payload?  I would argue that we don't even need to worry about versioning the endpoint if we do this because it's just an admin endpoint.

Reply:
I am also not 100% sure what the endpoint is doing. I think it is intended for the GPAlloc projects, to re-register them after they've been put back in the pool.

It arguably would be reasonable to do that. I would, however, probably push that we should version it regardless, both out of good practice, and since we would otherwise need to make changes in every automated script that uses it, or they would end up broken.

Reply:
(The change here was @marctalbott's idea, thus why I didn't go trying to fully understand it)

Reply:
Maybe let's put a comment on this method then saying that we believe this endpoint is primarily used by gpalloc.  The points I would want to emphasize are:

1.  Since we think gpalloc is the only user of this, that is why the Billing Account is hard coded.
2.  When we complete https://broadworkbench.atlassian.net/browse/CA-1104 we can investigate whether we should also delete this method.
3.  IF there are other use cases for this endpoint outside of gpalloc, we should update this comment to identify them